src/pc/mod.rs

```diff
@@ -76,6 +76,8 @@ pub(crate) struct ParachainCommanderOptions {
 	/// Defines subscription mode
 	#[clap(short = 's', long = "subscribe-mode", default_value_t, value_enum)]
 	pub subscribe_mode: SubxtSubscriptionMode,
+	#[clap(long = "last-skipped-slots", default_value = "10")]
```

Comment:
Needs doc comment to describe what the parameter does. This will show up in `--help`.